corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditorMain.java

```diff
@@ -95,21 +96,70 @@ public static void main(String[] args) {
             throw t;
         }
     }
-        public static int mainMethod(String[] args) {
-            CorfuRuntime runtime;
-            log.info("CorfuBrowser starting...");
-            // Parse the options given, using docopt.
-            Map<String, Object> opts =
-                    new Docopt(USAGE)
-                            .withVersion(GitRepositoryState.getRepositoryState().describe)
-                            .parse(args);
-            String host = opts.get("--host").toString();
-            Integer port = Integer.parseInt(opts.get("--port").toString());
-            boolean tlsEnabled = Optional.ofNullable(opts.get("--tlsEnabled"))
-                    .map(o -> Boolean.parseBoolean(o.toString()))
-                    .orElse(false);
+
+    private static String getOperation(Map<String, Object> opts) {
+        try {
             String operation = opts.get("--operation").toString();
+            return operation;
+        } catch (Exception e) {
+            return "";
+        }
+    }
+
+    private static String getOfflineDBDir(Map<String, Object> opts) {
+        try {
+            String path = opts.get("--offline-db-dir").toString();
+            return path;
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    private static String getHostName(Map<String, Object> opts) {
+        try {
+            String host = opts.get("--host").toString();
+            return host;
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    private static Integer getPortNumber(Map<String, Object> opts) {
+        final int defaultPort = 9000;
+        try {
+            Integer host = Integer.parseInt(opts.get("--port").toString());
+            return host;
+        } catch (Exception e) {
+            return defaultPort;
+        }
+    }
+
+    private static Boolean isTLSEnabled(Map<String, Object> opts) {
+        final Boolean defaultValue = false;
+        try {
+            Boolean tlsEnabled = Boolean.parseBoolean(opts.get("--tlsEnabled").toString());
+            return tlsEnabled;
+        } catch (Exception e) {
+            return defaultValue;
+        }
+    }
 
+    public static int mainMethod(String[] args) {
+        CorfuBrowserEditorCommands browser;
+        log.info("CorfuBrowser starting...");
+        // Parse the options given, using docopt.
+        Map<String, Object> opts =
+                new Docopt(USAGE)
+                        .withVersion(GitRepositoryState.getRepositoryState().describe)
+                        .parse(args);
+        String operation = getOperation(opts);
+        String offlineDbDir = getOfflineDBDir(opts);
+        String host = getHostName(opts);
+
+        Integer port = getPortNumber(opts);
+        boolean tlsEnabled = isTLSEnabled(opts);
+
+        if (host != null) {
```

Comment:
Can we add a comment here to say host passed in input args implies the tool wants to connect to an online corfu cluster?